CHANGES.rst

```diff
@@ -1,21 +1,22 @@
 Changes
 =======
 
-Version 0.9.0 (UNRELEASED)
+Version 0.9.0 (2022-12-13)
 --------------------------
 
 - Adds support for Python 3.11.
 - Adds support for Rucio.
 - Adds REANA specification validation and loading logic from ``reana-client``.
 - Adds support for networking/v1 API to Kubernetes Python client.
-- Adds new ``/api/launch`` endpoint that allows running workflows from remote sources.
+- Adds OpenAPI specification support for ``launch`` endpoint that allows running workflows from remote sources.
+- Adds OpenAPI specification support for ``get_workflow_retention_rules`` endpoint.
 - Adds support for specifying ``slurm_partition`` and ``slurm_time`` for Slurm compute backend jobs.
 - Adds generation of Kerberos init and renew container's configuration.
 - Adds specification of ``retention_days`` in ``reana.yaml`` JSON schema.
 - Adds support for Unicode characters inside email body.
 - Changes REANA specification loading functionality to allow specifying different working directories.
+- Changes REANA specification to allow enabling Kerberos for the whole workflow.
```

Comment:
I think we should keep this consistent with  _Adds specification of ``retention_days`` in ``reana.yaml`` JSON schema._, as they make the same kind of changes to `reana.yaml` schema.

Maybe we can write something similar to _Adds support for specifying ``slurm_partition`` and ``slurm_time`` for Slurm compute backend jobs._ for both of them?